src/cli/cli.cpp

```diff
@@ -2001,12 +2001,13 @@ void Interpreter::ProcessPollPeriod(int argc, char *argv[])
 
     if (argc == 0)
     {
-        mServer->OutputFormat("%d\r\n", (otLinkGetPollPeriod(mInstance) / 1000)); // ms->s
+        mServer->OutputFormat("%d\r\n", otLinkGetPollPeriod(mInstance));
     }
     else
     {
         SuccessOrExit(error = ParseLong(argv[0], value));
-        otLinkSetPollPeriod(mInstance, static_cast<uint32_t>(value * 1000)); // s->ms
+        VerifyOrExit(!(value < OPENTHREAD_CONFIG_MINIMUM_POLL_PERIOD), error = OT_ERROR_PARSE);
```

Comment:
=> `VerifyOrExit(value >= OPENTHREAD_CONFIG_MINIMUM_POLL_PERIOD, error = OT_ERROR_INVALID_ARGS);`

Reply:
Ok, updated in the third commit